scripts/exgdas_atmos_analysis_wdqms.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -x
+
+# Input GSI diagnostic file containing inputs to wdqms.py
+CNVSTAT="${CDUMP}.t${cyc}z.cnvstat"
+
+# Input files from CNVSTAT fed to wdqms.py
+INPUT_LIST=("diag_conv_ps_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_t_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_q_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_uv_ges.${PDY}${cyc}.nc4")
+
+# Observation types being processed by wdqms.py
+OTYPES=(SYNOP TEMP MARINE)
+
+# debugging and verbose options for wdqms.py
+[[ ${DEBUG_WDQMS:-} == "YES" ]] && debug="-d"
+[[ ${VERBOSE_WDQMS:-} == "YES" ]] && verbose="-v"
+
+################################################################################
+echo "Begin job ${job:-}"
+
+#-------------------------------------------------------------------------------
+# Enter working directory
+cd "${DATA}" || ( echo "FATAL ERROR: Unable to cd ${DATA}, ABORT!"; exit 2 )
+
+#-------------------------------------------------------------------------------
+# Copy cnvstat file from COMIN to DATA, untar and gunzip input files for wdqms.py
+cp "${COMIN}/${CNVSTAT}" .
+rc=$?
+(( rc != 0 )) && ( echo "FATAL ERROR: Unable to copy ${diag_file} from ${COMIN}, ABORT!"; exit 2 )
+for file in "${INPUT_LIST[@]}"; do
+  tar -xzvf "${diag_file}" "${file}.gz"
+  rc=$?
+  (( rc != 0 )) && ( echo "FATAL ERROR: Unable to extract ${file}.gz from ${diag_file}, ABORT!"; exit 3 )
+  gunzip "${file}.gz"
+  rc=$?
+  (( rc != 0 )) && ( echo "FATAL ERROR: Unable to gunzip ${file}.gz, ABORT!"; exit 3 )
+done
+
+#-------------------------------------------------------------------------------
+# Loop over observation types and produce csv files
+for otype in "${OTYPES[@]}"; do
+  echo "Processing ... ${otype}"
+  ${WDQMSPY} -i ${INPUT_LIST[@]} -t "${otype}" -o "${DATA}" "${debug:-}" "${verbose:-}"
+  rc=$?
```

Comment:
The "${debug:-}" "${verbose:-}" in the command line above creates empty strings " "  "  "  at the end of the command line and the wdqms.py interpret them as recognized arguments
```
Processing ... SYNOP
12 + /lfs/h2/emc/da/noscrub/emily.liu/git/global-workflow-wdqms/ush/wdqms.py -i diag_conv_ps_ges.2024031112.nc4 diag_conv_t_ges.2024031112.nc4 diag_conv_q_ges.2024031112.nc4 diag_conv_uv_ges.2024031112.nc4 -t SYNOP -o /lfs/h2/emc/stmp/emily.liu/RUNDIRS/v163_wdqms/2024031112/gdas/v163_wdqms_gdaswdqms_12.121646799.cbqs01 '' ''
usage: wdqms.py [-h] [-i INPUT_LIST [INPUT_LIST ...]] [-t TYPE] [-o OUTDIR]
                [-d] [-v]
wdqms.py: error: unrecognized arguments:  
14 + rc=2
14 + ((  rc != 0  ))
14 + echo 'FATAL ERROR: wdqms.py failed to process observation type SYNOP; ABORT!'

```